node/nodeHelper.go

```diff
@@ -151,17 +150,6 @@ func CreateNode(
 ) (*Node, error) {
 	var err error
```

Comment:
it looks like this var declaration can also be removed

Reply:
Right, removed it